### ISSUE TITLE
FTBFS: QApplication.cpp fatal error: QQuickStyle: No such file or directory

### ISSUE THREAD

**Issue by BlueT** [cfsghost/brig#6]
1st case:
Install/build with system Qt

ENV:
bluet@Titania:~/local/V-PlaySDK$ lsb_release -a
No LSB modules are available.
Distributor ID:	Ubuntu
Description:	Ubuntu 16.04.2 LTS
Release:	16.04
Codename:	xenial
bluet@Titania:~/local/V-PlaySDK$ qmake -v
QMake version 3.0
Using Qt version 5.5.1 in /usr/lib/x86_64-linux-gnu
bluet@Titania:~/local/V-PlaySDK$ sudo apt install qt5-default qtquick1-5-dev qtquick1-5-dev-tools qtcreator qt5-qmake qtdeclarative5-dev qtquick1-5-dev qml-module-qtquick-controls qtmultimedia5-dev qtbase5-private-dev qml-module-qtquick-controls
bluet@Titania:~/workspace/brig-example-app$ npm install brig

Log:

[npm-debug.log.txt](https://github.com/cfsghost/brig/files/1117040/npm-debug.log.txt)

**Comment by BlueT** [comment]
2nd case:
with 5.9.0 with V-Play

~~~~ log
ENV:
bluet@Titania:~/local/V-PlaySDK$ lsb_release -a
No LSB modules are available.
Distributor ID:	Ubuntu
Description:	Ubuntu 16.04.2 LTS
Release:	16.04
Codename:	xenial
bluet@Titania:~/workspace/brig-example-app$ LANG=C PATH=~/local/V-PlaySDK/5.9/gcc_64/bin/:$PATH PKG_CONFIG_PATH=~/local/V-PlaySDK/5.9/gcc_64/lib/pkgconfig/ CPATH=/home/bluet/local/V-PlaySDK/5.9/gcc_64/include/QtCore/5.9.0/:/home/bluet/local/V-PlaySDK/5.9/gcc_64/include/:/home/bluet/local/V-PlaySDK/5.9/gcc_64/:/home/bluet/local/V-PlaySDK/5.9/Src/ qmake -v
QMake version 3.1
Using Qt version 5.9.0 in /home/bluet/local/V-PlaySDK/5.9/gcc_64/lib
bluet@Titania:~/workspace/brig-example-app$ LANG=C PATH=~/local/V-PlaySDK/5.9/gcc_64/bin/:$PATH PKG_CONFIG_PATH=~/local/V-PlaySDK/5.9/gcc_64/lib/pkgconfig/ CPATH=/home/bluet/local/V-PlaySDK/5.9/gcc_64/include/QtCore/5.9.0/:/home/bluet/local/V-PlaySDK/5.9/gcc_64/include/:/home/bluet/local/V-PlaySDK/5.9/gcc_64/:/home/bluet/local/V-PlaySDK/5.9/Src/ npm install brig
~~~~

Same error

Log:
[npm-debug.log.txt](https://github.com/cfsghost/brig/files/1117043/npm-debug.log.txt)

**Comment by BlueT** [comment]
BTW, does brig support linux? Or only Mac/darwin?

**Comment by cfsghost** [comment]
Yup, it has Linux support, but you have to install Qt development environment to build brig by yourself.

[Document of QQuickStyle](https://doc.qt.io/qt-5/qquickstyle.html) said that QQuickStyle was implemented since Qt 5.7, so I think it's version issue cause QQuickStyle header is not found if you are using too old version of Qt.

For 2nd case, there is known issue you might get is that Qt version is too new. Actually, I don't test brig on Qt 5.8+ yet, but according to user's report, brig cannot be built since Qt 5.8. I just attempt to fix this bug.

If you can not wait me to fix it, trying to use Qt 5.7 to build brig, it should work.

**Comment by polarathene** [comment]
Not sure if this will help, I've been looking into using QtQuick 2.7(Qt 5.7+) and with python it seems you need to have [an environment value to point out where this module resides](https://stackoverflow.com/a/37845134).
